Connected/Assets/Scripts/Components/Splitter.cs

```diff
@@ -0,0 +1,84 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class Splitter : GeneralComponent
+{
+    private float ratio;
+    public Wire secondPositive { get; set; }
+
+    public (float, GeneralComponent) CheckSplitter()
+    {
+        float resistance1 = 0.0f, resistance2 = 0.0f, totRes;
+        GeneralComponent combiner1, combiner2;
+
+        if (CheckConnection(positive) && CheckConnection(secondPositive)) {
+            (resistance1, combiner1) = CheckSplitWire(positive.positive);
+            (resistance2, combiner2) = CheckSplitWire(secondPositive.positive);
+        } else {
+            combiner1 = null; combiner2 = null;
+        }
+
+        // If combiner1 isnt null and combiner2 is null it will catch after or sign.
+        if (combiner1 == null || (combiner1 != combiner2)) {
+            // TODO: Add appropriate error message
+            return (0, null);
```

Comment:
This tuple is used in a lot of different places, maybe we could make it a private static constant in this class?

Reply:
Will fix!